### PR TITLE
users and groups page visual debt

### PR DESCRIPTION
**Pull request by TheMaja**
- initial commit
- whitespace between buttons + next to buttons added
- added whitespace around filter text fields
- Scrolling behavior: keeps the filters in view and scrolls the rest

Related issues: [please specify]

## Description:

What are you up to? Fill us in :)

## Screenshots/videos
**visual debt removed**
image

## I solemnly swear that:

- [x] My code follows the style guidelines of this project
- [x] I have merged the develop branch into my branch and fixed any merge conflicts
- [x] I have performed a self-review of my own code
- [ ] I have commented my code in hard-to-understand areas
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Review by Sbonelo01 — APPROVED** [review]

**Review by FaithMo — CHANGES_REQUESTED** [review]
Looks good, please also check the storybook

**Review by KhalidUmuzi — APPROVED** [review]

File: frontend/src/components/pages/UsersAndGroups/Presentation.jsx [hunk 1]
```diff
@@ -201,8 +219,8 @@ export default function Presentation({
           );
         })}
       </Grid>
-      <Grid item xs={6}>
-        <Paper variant="outlined" elevation={2}>
+      <Grid item xs={6} className={classes.root}>
```
Comment (sheenarbw) [inline frontend/src/components/pages/UsersAndGroups/Presentation.jsx]: what does root mean? root generally means the root element. The top-level element.

And why are some grid items named root and some split? 

and what does split mean? And why is is applied inconsistently? 

Remember that good code makes its intentions clear. One way to do that is by using meaningful names.

Please also run the code - the scroll behavior is weird. I think this inconsistency might be the source of the bug.

**Review by sheenarbw — CHANGES_REQUESTED** [review]
- There are pretty random names used
- The code is not responsive - it only works on screens of specific sizes
- The ticket requirements are not met - the filters are meant to stay in view as you scroll down, but they dont
- there is a bug: The right hand column scrolling behavior is very different to the left hand column

Reviewers: It's critical that you check that the code does what it is meant to do. When it comes to frontend code you need to actually run it and make sure it works! Read the issue and make sure we are hitting all the requirements

**Review by Sbonelo01 — CHANGES_REQUESTED** [review]
This looks good @TheMaja . Please address the review talking about responsiveness: https://github.com/Umuzi-org/Tilde/pull/634#pullrequestreview-1160541207

The page isn't mobile friendly: 
users-and-groups

**Comment by TheMaja** [comment]
> This looks good @TheMaja . Please address the review talking about responsiveness: [#634 (review)](https://github.com/Umuzi-org/Tilde/pull/634#pullrequestreview-1160541207)
> 
> The page isn't mobile friendly: users-and-groups

This page won't be frequently used for mobile that's not necessary,  remember the discussion we had on sprint review.

**Review by Sbonelo01 — APPROVED** [review]

**Review by KhalidUmuzi — APPROVED** [review]

**Review by sheenarbw — CHANGES_REQUESTED** [review]
Is this PR up to date? It doesn't fix the scrolling problem

**Review by FaithMo — APPROVED** [review]
scrolling fixed

**Review by KhalidUmuzi — APPROVED** [review]

**Review by sheenarbw — CHANGES_REQUESTED** [review]
The scrolling behaves quite strangely. Did you do that on purpose? It looks like a bug to me.

If I add a reasonable number of users to the users list and I scroll down then the filters stay in place for a little while and then randomly they start to move out of the screen. 

Look at the app bar (the blue thing at the top). Do you see how you can always see the app bar? We want the filters to be like that, always 100% visible. 

A way to do this is to have 2 elements. Maybe 2 divs:

```
<div>
     Put the filter here
     this div is not scrollable
</div>
<div>
   Put the people or teams here
   this div is scrollable. 
   make sure it fits the whole screen and works on different screen sizes
</div>
```

**Review by Ngoakor12 — CHANGES_REQUESTED** [review]
mostly good, some things I think would add some improvement:
- making filter input box full width
- making the two sections scroll independently just like the section in the `/code_review_dash` page

**Pull request closed without merging**